conans/client/build/cppstd_flags.py

```diff
@@ -263,11 +265,18 @@ def _cppstd_gcc(gcc_version, cppstd):
         v20 = "c++2a"
         vgnu20 = "gnu++2a"
 
+    if Version(gcc_version) >= "11":
+        v20 = "c++20"
+        vgnu20 = "gnu++20"
+        v23 = "c++23"
```

Comment:
Isn't this ``c++2b`` at the moment?

Reply:
both have the same effect, so doesn't matter
```
  -std=gnu++2b                Conform to the ISO 2023 C++ draft standard with GNU extensions (experimental and incomplete support).  Same as -std=gnu++23.
```

Reply:
Yes, effect is the same, but it is the way to indicate the compiler doesn't have full support for c++23, like the above:

```python
if Version(gcc_version) >= "8":
        v20 = "c++2a"
        vgnu20 = "gnu++2a"
```

Shouldn't we follow the same approach as above?

Reply:
seems like c++17, c++20 and c++23 are still incomplete according to the https://gcc.gnu.org/projects/cxx-status.html

Reply:
I've changed `c++23` -> `c++2b` and `c++20` -> `c++2a` as they are incomplete.
not sure if I should touch `c++17`, it's still incomplete, but we already use that instead of `c++1z` for GCC 10.